src/Stratis.Bitcoin.Features.Miner/MinerSettings.cs

```diff
@@ -80,6 +102,11 @@ public MinerSettings(NodeSettings nodeSettings)
 
             this.BlockDefinitionOptions = new BlockDefinitionOptions(blockMaxWeight, blockMaxSize).RestrictForNetwork(nodeSettings.Network);
 
+            this.EnableCoinStakeSplitting = config.GetOrDefault("enablecoinstakesplitting", true, this.logger);
+            this.MinimumSplitCoinValue = config.GetOrDefault("minimumsplitcoinvalue", MinimumSplitCoinValueDefaultValue, this.logger);
+            this.MinimumStakingCoinValue = config.GetOrDefault("minimumstakingcoinvalue", MinimumStakingCoinValueDefaultValue, this.logger);
+            this.MinimumStakingCoinValue = this.MinimumStakingCoinValue == 0 ? 1 : this.MinimumStakingCoinValue;
```

Comment:
We are not allowing user to disable min utxo staking value?

Reply:
Yeah. But if you don't know the code and you want to disable it you will set it to 0 and expect it to be disabled, not to be reset to 1 against your will. 

There is no info in the help message that will mention that value of 0 is not allowed and will be overwritten to 1.

Reply:
Let's discuss this on slack or Skype, I don't understand your point quick

Reply:
Kk.